### ISSUE TITLE
Testing Big GML files

### ISSUE THREAD

**Issue by bermud** [opengeospatial/ets-gml32#22]
Seems that testing Big GML files is a problem 100 MBs or larger.

Discussed in OGC CITE SC Meeting - Washington TC 2016

**Comment by bermud** [comment]
Forum discussions related to this topic:
- http://stackoverflow.com/questions/40663/validating-a-huge-xml-file
- http://stackoverflow.com/questions/9788868/how-to-validate-big-xml-against-xsd-schema
- https://dzone.com/articles/splitting-large-xml-files-java

**Comment by bermud** [comment]
Next steps might be;

- implement a utility class to split up very large GML documents (GmlDocumentSplitter). 
- experiment with various chunk sizes to determine what works best (e.g. if doc size > 100 MB, split it into 50 MB chunks) 
- if necessary, do Schematron validation of the resulting chunks in parallel
